notifier/__main__.py

```diff
@@ -1,5 +1,37 @@
-from .core.notifierapp import NotifierApp
+import argparse
+from .core.celeryapp import CeleryApp
+from .core.flaskapp import FlaskApp
+
+
+def args():
```

Comment:
This options must be available from the config file.
A command line (who override the config file options) is pretty cool idea